TRADEOFFS.md

```diff
@@ -82,7 +81,10 @@ _m._ When splitting left and right multi-character pads, Lodash truncates from t
 Just truncates from the inside:\
 `just-left-pad('cde', 4, 'ab')` yields `'bcde'`\
 _n._ Lodash works with just one argument, Just expects two\
-_o._ Lodash treats `NaN` values as equal to one other 
+_o._ Lodash treats `NaN` values as equal to one other\
+_p._ Lodash accepts 3rd parameter for a default value.\
+_q._ just-get and just-set follows dotty for `(obj, ['a.b'])` style arguments. Lodash uses its own rules.\
```

Comment:
Don't know if this should be kept in fact, as the "array of strings" mostly works (except the quirk with references described below)?

Reply:
Difference is here: https://github.com/angus-c/just/issues/82

Reply:
Thx for the link.
BTW. What do you think about supporting the optional third parameter for `get` to be closer to lodash (almost a drop-in replacement)? It seems to be it would not even augment the size of the package, because the early returns are already there, and the only thing to change would be returning `defaultVal` instead of `void 0`.

Reply:
Sure, want to add it to the PR?

Reply:
Ok, will work on it 👍